library/weno_log_sync.php

```diff
@@ -7,12 +7,41 @@
  *  @copyright Copyright (c) 2021 Sherwin Gaddis <sherwingaddis@gmail.com>
  *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
+use OpenEMR\Common\Crypto\CryptoGen;
 
-use OpenEMR\Rx\Weno\Container;
+use OpenEMR\Modules\WenoModule\Services\LogProperties;
+use OpenEMR\Modules\WenoModule\Services\WenoPharmaciesJson;
 
+
+if(isset($_GET['key']) && !empty(isset($_GET['key']))){
+    start_weno();
+}
 function start_weno()
 {
-    $container = new Container();
-    $logsync = $container->getLogproperties();
+    $logsync = new LogProperties();
+    $logsync->logSync();
+}
+
+function downloadWenoPharmacy(){
+    $cryptoGen = new CryptoGen();
+    $localPharmacyJson = new WenoPharmaciesJson(
+        $cryptoGen
+    );
```

Comment:
tip here.
You could simplify this be removing the $cryptoGen parameter from the class above and simply creating and setting it in the constructor. (then no need to create the CryptoGen class here or need to have it in above use ... statement)